app/namespace.go

```diff
@@ -638,6 +645,10 @@ func NewNamespaceCommand(getNamespaceClientFn GetNamespaceClientFn) (CommandOut,
 					}
 				}
 
+				n.Spec.Lifecycle = &namespace.LifecycleSpec{
```

Comment:
the default behaviour (i.e. without specifying the flag) is that delete protection is disabled, right?

Reply:
yes tested it is disabled by default